lib/fontist/index.rb

```diff
@@ -2,6 +2,8 @@
 
 module Fontist
   class Index
+    VERSION = 2
```

Comment:
Thinking of storing an index at user's system to simplify working with different versions. Haven't chosen yet. Will do this, or new version as a new file.

Reply:
Yes let’s do it at “~/.fontist/formulas/“?

Reply:
👍 👌